language/tools/vm-genesis/src/lib.rs

```diff
@@ -119,6 +122,7 @@ pub fn encode_genesis_change_set(
 
     let move_vm = MoveVM::new();
     let mut session = move_vm.new_session(&data_cache);
+    let logger = NoLogLogger;
```

Comment:
should genesis get a stderror logger just for good luck?

Reply:
Yeah I think it's a good idea to use stderr here as the information could help us understand what goes wrong when debugging. The bottom line is, this is an internal tool we use ourselves so I'd personally not worry too much about this spamming the out.